### DISCUSSION TITLE
Where to find the hyper functions after installing toolkit?

### DISCUSSION THREAD

**Discussion by ngocvo3103** [timescale/timescaledb-toolkit#311] [Q&A] [answered]
Hi folks, 

After installing the toolkit successfully, I restarted the timescaledb instance and try using the hyper functions, but still got the error saying that the function is not there. Is there anything else that I should do in order to use the hyper functions?

Thank you for helping me out.

```
Error executing SELECT statement. ERROR: function stats_agg(integer) does not exist
  Hint: No function matches the given name and argument types. You might need to add explicit type casts.
  Position: 416 - Connection: [SIT] iperfdb: 237ms
```

**Comment by ngocvo3103** [comment, marked as answer]
Found it! You need to run the below command as super user in order to activate the hyper functions:

`CREATE EXTENSION timescaledb_toolkit`

**Reply by davidkohn88** [reply]
Thanks! Yes that's the answer! Sorry we didn't get to it sooner!
